sdk/appcenter-push/build.gradle

```diff
@@ -4,6 +4,8 @@ evaluationDependsOn(':sdk')
 dependencies {
     api project(':sdk:appcenter')
 
+    //noinspection GradleDependency cannot update this without sync error, but app can update
     api "com.google.firebase:firebase-core:16.0.1"
+    //noinspection GradleDependency cannot update this without sync error, but app can update
```

Comment:
Is this something that we need to document?  It could be confusing for people to see AC uses an older firebase version and then not knowing if they can update.

Reply:
No it's not documented, it should be a separate task though, this comment is not adding anything new compared to master and this PR needs to be merged asap as develop broken.